dev/com.ibm.websphere.appserver.jpa-2.1/com.ibm.websphere.appserver.jpa-2.1.feature

```diff
@@ -226,8 +226,8 @@ Subsystem-Name: Java Persistence API 2.1
 -features=com.ibm.websphere.appserver.jndi-1.0, \
  com.ibm.websphere.appserver.transaction-1.2, \
  com.ibm.websphere.appserver.jdbc-4.1; ibm.tolerates:="4.2", \
- com.ibm.websphere.appserver.jpaContainer-2.1
--bundles=com.ibm.websphere.appserver.thirdparty.eclipselink; location:=dev/api/third-party/, \
- com.ibm.ws.jpa.container.eclipselink
+ com.ibm.websphere.appserver.jpaContainer-2.1, \
+ com.ibm.websphere.appserver.org.eclipse.persistence-2.6
+-bundles=com.ibm.ws.jpa.container.eclipselink
```

Comment:
is the `-bundle` dep still needed?  We should be getting this bundle through the `om.ibm.websphere.appserver.org.eclipse.persistence-2.6` feature now

Reply:
The -bundle brings in the eclipselink container integration, that's part of the JEE JPA Container eclipselink integration, and should not be part of the persistence-2.6 feature (which is only supposed to bring in the eclipselink jpa provider impl binary)

So it is still needed.

Reply:
ah yes you're right.  I thought this was the actual eclipselink bundle.